package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "vscode-extension-telemetry-wrapper",
-  "version": "0.14.0",
+  "version": "0.14.1",
```

Comment:
how about 0.15.0?  vscode-telemetry lib changed from 0.9.6 to 1.2.0, theoretically there might even be breaking changes there.

Reply:
updated, please review again.
The major version update of the vscode-telemetry package did indeed change the previously exported types, and this change has already been fixed in the [PR](https://github.com/Eskibear/vscode-extension-telemetry-wrapper/pull/53).